File: WebTool-Web/src/main/webapp/js/controller/bhVersionService.js
//服务层
app.service('bhVersionService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../bhVersion/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../bhVersion/findPage.do?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../bhVersion/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../bhVersion/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../bhVersion/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../bhVersion/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../bhVersion/search.do?page='+page+"&rows="+rows, searchEntity);
	}    	
});

File: WebTool-Web/src/main/java/com.lxl.webtool/dao/BhVersionService.java
package com.lxl.webtool.dao;

import com.lxl.webtool.dao.pojo.TbBhVersion;
import com.lxl.webtool.pojo.PageResult;

import java.util.List;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface BhVersionService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbBhVersion> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum, int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(TbBhVersion bhVersion);
	
	
	/**
	 * 修改
	 */
	public void update(TbBhVersion bhVersion);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public TbBhVersion findOne(Long id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Long[] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(TbBhVersion bhVersion, int pageNum, int pageSize);
	
}
